rust/datafusion/src/physical_plan/string_expressions.rs

```diff
@@ -66,3 +71,73 @@ pub fn concatenate(args: &[ArrayRef]) -> Result<StringArray> {
     }
     Ok(builder.finish())
 }
+
+/// character_length returns number of characters in the string
+/// character_length('josé') = 4
+pub fn character_length(args: &[ArrayRef]) -> Result<Int32Array> {
```

Comment:
Any reason to use `Int32` instead of `UInt32`?

Reply:
Agree and fixed.